client/jetpack-connect/auth-logged-in-form.jsx

```diff
@@ -108,6 +108,12 @@ class LoggedInForm extends Component {
 			if ( ! isRedirectingToWpAdmin && authorizeSuccess ) {
 				return this.props.goBackToWpAdmin( queryObject.redirect_after_auth );
 			}
+		} else if ( props.isWCS ) {
```

Comment:
Why not add this condition to the existing `if ( props.isSSO ) {` statement since the body is the same?